main/medium/src/misc_newtypes.rs

```diff
@@ -1118,6 +1118,65 @@ impl TryFrom<f64> for PositionInQuarterNotes {
     }
 }
 
+/// This represents a position expressed as an amount of midi ticks (PPQ).
+///
+/// Can be negative, see [`PositionInSeconds`](struct.PositionInSeconds.html).
+#[derive(Copy, Clone, PartialEq, PartialOrd, Debug, Default, Display)]
+#[cfg_attr(
+    feature = "serde",
+    derive(Serialize, Deserialize),
+    serde(try_from = "f64")
+)]
+pub struct PositionInPPQ(pub(crate) f64);
```

Comment:
```suggestion
pub struct PositionInPpq(pub(crate) f64);
```

Reply:
Already Got this naming convention. I don't like much CamelCase for abbreviations...